packages/web-components/src/global/settings.scss

```diff
@@ -0,0 +1,9 @@
+@use "sass:map";
+@use "uswds-core" with ( 
+    $theme-body-font-size: 10
+);
+
+// Sass maps are immutable, so we need to re-assign the original variable
+// after setting it
+uswds-core.$all-color-shortcodes:  map.set(uswds-core.$all-color-shortcodes, "error-dark", #cd2026);
```

Comment:
After our conversation in standup I went back to do some more discovery and found that this pattern allows us to set the tokens to be whatever we want and avoids the USWDS validation of them. However, I don't recommend this for regular use since it skips some intermediary steps (the `$tokens-color-state` map isn't updated, for example). But this could be used as a stepping stone to get us to USWDS3 while avoiding dealing with color changes, and then once we're ready we could remove it.

cc @k80bowman @humancompanion-usds